recipes-webadmin/cockpit-config/files/cockpit

```diff
@@ -0,0 +1,2 @@
+Cmnd_Alias    NOTTYCMDS = /usr/bin/cockpit-bridge --privileged
```

Comment:
`cockpit` is not a meaningful name for this file in the yocto directory
I propose `sudoers-rules` or `sudoers-rules-cockpit`.

But keep the name `cockpit` when you copy it on the target

Reply:
I follow files name already defined in the `sudoers.d` dir:
```yoctoCI:~$ ls /etc/sudoers.d
admin	      adminsys	cockpit     maint-n1  operator	  snmp
admincluster  ansible	emergadmin  maint-n3  privileged
```